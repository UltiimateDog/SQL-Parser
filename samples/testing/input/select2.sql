SELECT * FROM Student WHERE Student.B = 100;