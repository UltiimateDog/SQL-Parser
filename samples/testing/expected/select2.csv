3, 100, 105, 44
4, 100, 50, 11
5, 100, 500, 22
